target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -40,6 +40,7 @@ hiwifi,hc5761a)
 mediatek,linkit-smart-7688)
 	ucidef_set_led_wlan "wifi" "wifi" "linkit-smart-7688:orange:wifi" "phy0tpt"
 	;;
+netgear,r6080|\
```

Comment:
Since you have a trigger in DTS now, you have to create a separate case for r6080 (you may join them later again when you remove the wifi for r6120).

Reply:
Created separate case in the R6080 commit, and then merged them again in the R6120 commit, after the R6120 triggers get moved.